blog/docs/releases/announcing-knative-v1-5-release.txt

```diff
@@ -0,0 +1,205 @@
+---
+title: "1.5 release"
```

Comment:
This doc also needs to be added to the blog navigation: https://github.com/knative/docs/blob/main/blog/config/nav.yml#L16

Reply:
Doc should also be named `...release.md`